### PR TITLE
Simplify configuration of `containerize-dependencies` goal

### PR DESCRIPTION
**Pull request by epwilkins**

### CODE REVIEW

File: habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/ContainerizeDepsMojo.java [hunk 1]
```diff
@@ -162,74 +101,65 @@ protected void doExecute() throws MojoExecutionException, MojoFailureException {
...
-        );
-        virtualEnvironmentPath = HabushuUtil.getCleanVirtualEnvironmentPath(virtualEnvironmentPath);
-
-        // if a valid virtual environment was found, relativize and add to the fileSet for exclusions
-        if (virtualEnvironmentPath != null && !virtualEnvironmentPath.isEmpty()
-                && new File(virtualEnvironmentPath).isDirectory()) {
-            Path venvDirRelativeToBasedir = getWorkingDirectoryPath(project)
-                    .relativize(Paths.get(virtualEnvironmentPath));
-            fileSet.addExclude(venvDirRelativeToBasedir + GLOB_RECURSIVE_ALL);
-        }
```
Comment (d-ryan-ashcraft) [inline habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/ContainerizeDepsMojo.java:232]: S: I'm good with the rest of this change, but not good w/ the hardcoded paths.  I just don't think we can let you customize these in the rest of habushu and then force defaults here.

Comment (epwilkins) [reply habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/ContainerizeDepsMojo.java:232]: Wouldn't you still be able to customize them?  You'd just provide the alternative like this:
```xml
                <configuration>
                    <defaultSourceSet>
                        <excludes>
                            <exclude>my-build-dir/**</exclude>
                        </excludes>
                    </defaultSourceSet>
                </configuration>
```

Comment (d-ryan-ashcraft) [reply habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/ContainerizeDepsMojo.java:232]: Let me rephrase.  If I've already customized them in habushu, I now have to manually do it again.  When we support a variable for path A in the plugin in one place, it seems incongruent to not respect it in another.

Comment (d-ryan-ashcraft) [reply habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/ContainerizeDepsMojo.java:232]: Synced w/ Emily offline - I think there is a middle ground approach here that I'll provide when I have a couple.
